sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 ### New Features
 
 - Added support for `CreateIfNotExists` for Share and Directory clients, and `DeleteIfExists` for Share, Directory and File clients.
+- Support setting file sas permission with a raw string.
```

Comment:
Should this be `files shares` or is `file` more accurate and grammatically correct?
```suggestion
- Support setting file SAS permission with a raw string.
```